_config.yml

```diff
@@ -5,3 +5,5 @@ exclude:
 
 gems:
   - jekyll-redirect-from
+
+baseurl: /homepage
```

Comment:
Would not this break this project? If I add `/homepage` in the currently generated page it doesn't work.